src/app/panels/terminal.js

```diff
@@ -45,7 +45,7 @@ class Terminal extends Plugin {
     self._api = api
     self._opts = opts
     self.data = {
-      lineLength: opts.lineLength || 80,
+      lineLength: opts.lineLength || 80, // ???
```

Comment:
Not sure why you added `????` 
This mean that he lineLength has he default value of 80, and can be override by the opts.

Reply:
@GrandSchtroumpf  because opts has no lineLength. How it should be used?

Reply:
ah ... I don't know XD. Maybe add a comment a little bit more descriptive than "???" then ^^